### ISSUE TITLE
feat: replace fixed stoploss percentage with ATR-normalized stops for all positions

### ISSUE THREAD

**Issue by Aviksaikat** [tradingstrategy-ai/gmx-ccxt-freqtrade#22]
Part of #11

## Problem

The current fixed percentage stoploss (`-0.129` for V2 longs, `-0.2` for V3 longs) is applied uniformly across all pairs regardless of volatility. A 12.9% stop on ETH (ATR ~3%) is functionally very different from 12.9% on a degen coin (ATR ~8%). This causes:

- **Over-wide stops** on low-volatility blue chips (money left on the table on losers)
- **Premature stops** on high-volatility alts during normal noise
- V2 currently bleeds **-328 USDT** across 186 long stop_loss exits

### Long side
Replace fixed `stoploss` with `stoploss_from_absolute()` based on entry ATR:
```python
stop_distance = long_atr_stop_multiplier * entry_atr
stoploss = stoploss_from_absolute(trade.open_rate - stop_distance, trade.open_rate)
```

### Velocity exit threshold (shorts)
Normalise `velocity_tp_threshold` by ATR instead of raw percentage:
```python
velocity_exit = price_velocity_pct / (entry_atr / entry_price)
```

## Experiment design

Three arms:
- **(A)** Current fixed stops (baseline)
- **(B)** ATR-normalized long stops only
- **(C)** ATR-normalized long stops + ATR-normalized velocity threshold

## Checklist

- [ ] ATR long stop implemented via `stoploss_from_absolute()` in V2 and V3
- [ ] ATR-normalized velocity threshold implemented (optional, separate flag)
- [ ] Experiment run across all 3 arms on canonical time range
- [ ] No regression on Sharpe vs baseline
